src/test/scala/uk/gov/hmrc/scalatestaccessibilitylinter/domain/KnownIssuesSpec.scala

```diff
@@ -48,10 +48,11 @@ class KnownIssuesSpec extends AnyFeatureSpec with Matchers with DiffMatcher {
 
   Feature("can load known issues successfully from config") {
     Scenario("uk/gov/hmrc/scalatestaccessibilitylinter/accessibility-linter.conf") {
-      knownIssues should matchTo(
+      knownIssues.knownIssues.length               should be(26)
```

Comment:
this now runs against the real config, so just tweaked the existing test to check the number of entries is as expected, and match the first couple in detail